### PR TITLE
Cleanup content page

### PR DESCRIPTION
**Pull request by DrKlonk**
Removed some unused code which was replaced by Algolia

### CODE REVIEW

**Comment by DrKlonk** [comment]
The failing test is fixed in the other branch

File: frontend/pages/content/index.vue [hunk 1]
```diff
@@ -65,74 +55,13 @@ export default defineComponent({
...
-        })
-      } catch (e) {
-        console.log(e)
-      } finally {
-        isLoading.value = false
-      }
-    }
     const algoliaIndex: string = process.env.algoliaIndex || ''
     const appId: string = process.env.algoliaAppId || ''
     const searchKey: string = process.env.algoliaSearchKey || ''
```
Comment (jeroendrenth) [inline frontend/pages/content/index.vue:67]: This only works when I enter the Admin API Key. Is this the correct key to use when searching?

**Comment by jeroendrenth** [comment]
Nothing happens when the api keys are not correct. Maybe add a generic error message. We can do this in a separate user story

**Comment by jeroendrenth** [comment]
> The failing test is fixed in the other branch

Maybe merge that one into this branch

File: frontend/pages/content/index.vue [hunk 2]
```diff
@@ -164,6 +91,7 @@ export default defineComponent({
     display: flex;
     justify-content: center;
   }
+
   &__button {
```
[the remark this answers is not in the document]
Comment (DrKlonk) [reply frontend/pages/content/index.vue:95]: check

File: frontend/pages/content/index.vue [hunk 3]
```diff
@@ -65,74 +55,13 @@ export default defineComponent({
     AisInfiniteHits,
     AisSearchBox,
     ClipLoader,
-    BaseButton,
   },
 
   setup() {
-    const { store } = useContext()
-
     const limit = 6
```
[the remark this answers is not in the document]
Comment (DrKlonk) [reply frontend/pages/content/index.vue:61]: yes

File: frontend/pages/content/index.vue (hunk 1 above)
Comment (DrKlonk) [reply frontend/pages/content/index.vue:67]: Are you sure? I'm using my search key and it works fine

Comment (jeroendrenth) [reply frontend/pages/content/index.vue:69]: No, think I was wrong

**Pull request merged**